doc/CONTRIBUTING.md

```diff
@@ -106,6 +108,13 @@ Don't worry about setting DynamoDB endpoints or creating tables: the broker's co
 
 **Note:** The local version of DynamoDB is not recommended for production.
 
+### Install RabbitMQ
+
+RabbitMQ is used to pass jobs to the validator, and requires Erlang to be installed before RabbitMQ.
+
+1.  Install Erlang based on the [download instructions](https://www.erlang.org/downloads)
+2.  Choose an installation guide based on your OS for [RabbitMQ](https://www.rabbitmq.com/download.html).  Be sure to install Erlang before installing RabbitMQ.  The default user and password is "guest"/"guest", be sure to update the config.yml file if you change this.
```

Comment:
Changed the example values to guest/guest, and updated the text referring to the config file.  In the typical case the user will not be changing those and so doesn't need to pay attention to it in the config file (it's actually complicated to change them).  RabbitMQ actually runs immediately after installation, no extra steps are needed.  Stopping and restarting the service can be more complicated, the information is available in the install guide we link to if they want to do that, but the typical user just needs to install and move to the next step.